clients/ethereumj:develop/harmony.sh

```diff
@@ -99,16 +103,14 @@ echo "Loading initial blockchain..."
 if [ -f /chain.rlp ]; then
     export HARMONY_ETHER_CAMP_OPTS="$FLAGS -Dblocks.format=rlp -Dblocks.loader=/chain.rlp -Dpeer.listen.port=0"
     echo "importBlocks options: $HARMONY_ETHER_CAMP_OPTS"
-    eval "${HARMONY_EXECUTABLE} importBlocks"
+    eval ${HARMONY_EXECUTABLE}
 fi
 
 # Load the remainder of the test chain
 if [ -d /blocks ]; then
     echo "Loading remaining individual blocks..."
-    for block in `ls /blocks | sort -n`; do
-        export HARMONY_ETHER_CAMP_OPTS="$FLAGS -Dblocks.format=rlp -Dblocks.loader=/blocks/$block -Dpeer.listen.port=0"
-		eval "${HARMONY_EXECUTABLE} importBlocks"
-	done
+    export HARMONY_ETHER_CAMP_OPTS="$FLAGS -Dblocks.format=rlp -Dblocks.loader=/blocks -Dpeer.listen.port=0"
+    eval ${HARMONY_EXECUTABLE}
```

Comment:
I've seen this before, and am a bit curious. What's the benefit in using `eval` for executing a command? None of the other client-script uses that.

Reply:
It's a deprecated part of code. I've replaced this command with `run_harmony` function already.